src/sentry/processing/backpressure/redis.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any, Generator, Mapping, Sequence, Union
+
+import rb
+from redis import Redis
+from rediscluster import RedisCluster
+
+# Based on configuration, this could be:
+# - a `rb` (aka redis blaster) Cluster
```

Comment:
I would only support redis and rediscluster, not rb. if we want to monitor a rb instance, we should take the opportunity and move it to a different setup